python/meterpreter/meterpreter.py

```diff
@@ -22,14 +22,6 @@
 else:
 	has_windll = hasattr(ctypes, 'windll')
 
-# this MUST be imported for urllib to work on OSX
```

Comment:
Yes, this was related to rapid7/metasploit-framework#5013 and rapid7/metasploit-framework#5220. According to the description it's for Python 2.7.6 on OSX 10.10.